build.assets/protos-up-to-date.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Detects whether protobufs are up to date.
+#
+
+CUR_DIR="$(dirname "$0")"
```

Comment:
In terms of bash style, I would suggest having a `main` function and using local-scoped variables as much as possible. See genproto.sh itself for an example - https://github.com/gravitational/teleport/blob/master/build.assets/genproto.sh.

This is a small, simple script, but the discipline carries over to larger pieces (or so I hope).

Reply:
I find this to be a reasonably good source, in lack of something more "official": https://www.netmeister.org/blog/writing-shell-scripts.html.